doc/build-unix.md

```diff
@@ -142,6 +142,26 @@ libqrencode (optional) can be installed with:
 
     sudo dnf install qrencode-devel
 
+Dependency Build Instructions: Arch Linux
+-------------------------------------
+Build requirements:
+
+sudo pacman -S gcc-c++ libtool make autoconf automake openssl-devel libevent-devel boost-devel libdb4-devel libdb4-cxx-devel zeromq binutils m4 make pkg-config miniupnpc colorcore
+
+opcional libs:
```

Comment:
All proposed seems optional, but withdrew these requirements of the official documentation of the AUR, the very archlinux wiki has little on the Bitcoin-core.
I see that such membership can better bitcoin project because the arch has many forks that can use this basic documentation.